### PR TITLE
1 initial setup

### LINKED ISSUE #1: Initial project setup - Radio app
* .NET solution file
* .NET projects (radio app)
  * User Interface
  * Backend Interface
  * Hardware Abstraction Layer
* High-Level Documentation

### PR DESCRIPTION
**Pull request by zjcers**
Adds initial project files and documentation. By no means is this final material, but this should be enough for us to get started. resolves #1

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
```
Comment (jackdweber) [inline .vscode/launch.json:2]: Are comments like this okay in JSON, github highlights them red

**Review by jackdweber — COMMENTED** [review]
Look at json thing, then i think everyone should check out this PR, so they all know whats going on specifically.

**Review by jackdweber — DISMISSED** [review]
nvm i googled, and JSON does not support comments.

File: docs/features/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* [Provisioning mode](../features/ProvisioningMode.md)
+
+
+**Optional**
+
+
+
+Software - Management Console/Backend
+-----------------------------
+
```
Comment (johnying7) [inline docs/features/README.md:46]: split management console frontend and backend

**Comment by zjcers** [comment]
launch.json isn't normal JSON. It's really a VSCode config file. I didn't even write those comments, code put them in there automatically.

File: .vscode/launch.json (hunk 1 above)
Comment (zjcers) [reply .vscode/launch.json:2]: Tried to comment on this using my phone, but this text box doesn't work in MobileSafari. This way fine the way it is, in fact, it came that way. The reason is is that this isn't true JSON, but instead a VS Code-flavored variant that supports comments. The comments actually came from the default launch.json.

**Review by johnying7 — APPROVED** [review]

**Review by aamirani — APPROVED** [review]
Good to go once we have the management console frontend and backend split up.

**Pull request merged**